docs/source/langevin_sde.rst

```diff
@@ -0,0 +1,26 @@
+Stochastic Differential Equations (SDE) for Langevin dynamics
+=============================================================
+
+The general Langevin dynamics can be described by the following SDE:
+
+.. math::
+
+    d c_{i} = -\lambda\partial_{i} f(\mathbf{c}) + \sigma dW_{i}\;\;\text{with}\;\;c_{i}(0)=0\;\;\forall i=1,\ldots,N
+
+where :math:`\lambda` and :math:`\sigma` are hyper-parameters controlling the strengths of the drift and diffusion terms, respectively. We have implemented this solver as a classical solver implemented solely on classical computer, that is, there is no optical simulation of the Langevin dynamics in our ``ccvm`` package.
```

Comment:
In the paper, we use "hyperparameters" (no hyphen).

I think it's odd to say "implemented" twice, but I'm not too fussed about it.

If by "the Langevin dynamics", you're talking about a solver, maybe that should be stated. Otherwise, I'm not sure why it wouldn't just read "Langevin dynamics" (without "the").

Reply:
I fixed the issues other than "implemented x2" (I was worried about my edit attempt changing the meaning of the sentence).